### PR TITLE
sci-libs/geos: version bump to 3.6.0

### PR DESCRIPTION
**Pull request by P424D0X**

### CODE REVIEW

File: sci-libs/geos/geos-3.6.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+src_compile() {
+	emake
+
+	use doc && emake -C "${S}/doc" doxygen-html
+}
+
+src_install() {
+	emake DESTDIR="${D}" install
```
Comment (SoapGentoo) [inline sci-libs/geos/geos-3.6.0.ebuild]: `default` instead

File: sci-libs/geos/geos-3.6.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+	use doc && dohtml -r doc/doxygen_docs/html/*
```
Comment (SoapGentoo) [inline sci-libs/geos/geos-3.6.0.ebuild]: `dohtml` is deprecated in `EAPI=6`

File: sci-libs/geos/geos-3.6.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+	use python && python_optimize "${D}$(python_get_sitedir)"/geos/
+
+	find "${ED}" -name '*.la' -exec rm -f {} +
```
Comment (SoapGentoo) [inline sci-libs/geos/geos-3.6.0.ebuild]: use `find "${D}" -name '*.la' -delete || die` instead

File: sci-libs/geos/geos-3.6.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+src_configure() {
+	econf \
+		$(use_enable python) \
+		$(use_enable ruby) \
+		$(use_enable php) \
+		$(use_enable static-libs static)
+}
+
+src_compile() {
+	emake
```
Comment (SoapGentoo) [inline sci-libs/geos/geos-3.6.0.ebuild]: default

File: sci-libs/geos/geos-3.6.0.ebuild [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+"
+DEPEND="${RDEPEND}
+	doc? ( app-doc/doxygen )
+	php? ( dev-lang/swig )
+	python? ( dev-lang/swig ${PYTHON_DEPS} )
+	ruby? ( dev-lang/swig )
+"
+
+src_prepare() {
+	epatch "${FILESDIR}"/3.4.2-solaris-isnan.patch
```
Comment (SoapGentoo) [inline sci-libs/geos/geos-3.6.0.ebuild]: put the patch in `PATCHES`, then call `default`

File: sci-libs/geos/geos-3.6.0.ebuild [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+	eautoreconf
+	echo "#!${EPREFIX}/bin/bash" > py-compile
```
Comment (SoapGentoo) [inline sci-libs/geos/geos-3.6.0.ebuild]: missing `|| die`

File: sci-libs/geos/geos-3.6.0.ebuild [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+	eapply_user
```
Comment (SoapGentoo) [inline sci-libs/geos/geos-3.6.0.ebuild]: after having called `default`, nix this

File: sci-libs/geos/geos-3.6.0.ebuild [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+PYTHON_COMPAT=( python2_7 )
+inherit autotools eutils python-single-r1 python-utils-r1
+
+DESCRIPTION="Geometry engine library for Geographic Information Systems"
+HOMEPAGE="http://trac.osgeo.org/geos/"
+SRC_URI="http://download.osgeo.org/geos/${P}.tar.bz2"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="amd64 arm ppc ppc64 x86 ~x64-freebsd ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~sparc-solaris ~sparc64-solaris ~x64-solaris"
```
Comment (SoapGentoo) [inline sci-libs/geos/geos-3.6.0.ebuild]: this contains stable `KEYWORDS`, which is not allowed

File: sci-libs/geos/geos-3.6.0.ebuild [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+IUSE="doc php python ruby static-libs"
+
+RDEPEND="
+	php? ( >=dev-lang/php-5.3:* )
+	ruby? ( dev-lang/ruby:* )
+"
+DEPEND="${RDEPEND}
+	doc? ( app-doc/doxygen )
+	php? ( dev-lang/swig )
+	python? ( dev-lang/swig ${PYTHON_DEPS} )
```
Comment (SoapGentoo) [inline sci-libs/geos/geos-3.6.0.ebuild]: `python-single-r1` in combination with `DEPEND`-only `${PYTHON_DEPS}` makes no sense

File: sci-libs/geos/geos-3.6.0.ebuild [hunk 10]
```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="6"
+
+PYTHON_COMPAT=( python2_7 )
+inherit autotools eutils python-single-r1 python-utils-r1
```
Comment (SoapGentoo) [inline sci-libs/geos/geos-3.6.0.ebuild]: don't need `python-utils-r1`, inherited by `python-single-r1` already

**Comment by a17r** [comment]
3.6.1 is now in tree. Please note that USE=php was dropped as it is now a separate upstream package.

**Pull request closed without merging**